server/lib/utils.js

```diff
@@ -197,6 +197,22 @@ const defaultHostId = () => {
 }
 
 /**
+ * Check if this is an internal email address.
+ * Useful for testing emails in localhost or staging
+ */
+const isEmailInternal = (email) => {
+  if (email.toLowerCase().indexOf('@opencollective.com') != -1 ||
+    email.toLowerCase().indexOf('@opencollective.org') != -1 ||
+    email.toLowerCase() === 'asood123@gmail.com' ||
+    email.toLowerCase() === 'piamancini@gmail.com' ||
+    email.toLowerCase() === 'xdamman@gmail.com' ||
+    email.toLowerCase() === 'sedubois@gmail.com') {
```

Comment:
Could we please avoid committing private emails in source code? If really needed, instead use opencollective.com emails?